actionpack/lib/action_dispatch/middleware/debug_exceptions.rb

```diff
@@ -66,7 +66,21 @@ def call(env)
       response
     rescue Exception => exception
       raise exception unless request.show_exceptions?
-      render_exception(request, exception)
+
+      # If we got to this point, the only thing that can crash is the
```

Comment:
Should we try and pare this down a little bit, to just the essential? I feel like if people want the whole story, we should look at the commit for context 😬 😬